fe/fe-core/src/main/java/org/apache/doris/analysis/SelectStmt.java

```diff
@@ -1966,6 +1971,16 @@ private boolean checkGroupingFn(Expr expr) {
         return false;
     }
 
+    private void getAggregateFnExpr(Expr expr, ArrayList<Expr> aggFnExprList) {
+        if (expr instanceof FunctionCallExpr && expr.fn instanceof AggregateFunction) {
+            aggFnExprList.add(expr);
+        } else if (expr.getChildren() != null && expr.getChildren().size() > 0) {
```

Comment:
remove else if (expr.getChildren() != null && expr.getChildren().size() > 0) call `            for (Expr child : expr.getChildren()) {
                getAggregateFnExpr(child, aggFnExprList);
            }` directly

Reply:
Thanks for the reminder, I have modified it, but I have kept `else if (expr.getChildren() != null)`, otherwise it may cause NullPointerException.